app/models/concerns/foreman_openscap/host_extensions.rb

```diff
@@ -63,7 +63,9 @@ def get_asset
     end
 
     def policies_enc
-      combined_policies.map(&:to_enc).to_json
+      check = ForemanOpenscap::OpenscapProxyAssignedVersionCheck.new(self).run
```

Comment:
The check in here checks for `<= 0.6.1` but 0.6.1 already has tailoring file API and digests. The comparing should be `< 0.6.1` I believe